random/random.mbti

```diff
@@ -3,13 +3,21 @@ package moonbitlang/core/random
 // Values
 
 // Types and methods
-type RandomState
-impl RandomState {
-  gen_double(Self) -> Double
-  gen_int(Self) -> Int
-  gen_int64(Self) -> Int64
-  init_state(~seed : Int = ..) -> Self
-  new() -> Self
+type ChaCha8
```

Comment:
why this is exposed

Reply:
It was removed in x , but out of sync here. Sorry